atd-vze/src/Components/GridExportData.js

```diff
@@ -98,7 +98,31 @@ const GridExportData = ({ query, columnsToExport, totalRecords }) => {
       return flattenedRow;
     });
 
-    return flattenedData;
+    const cleanedAndFlattenedData = flattenedData.map(item => {
+      // Some fields are presenting issues when they are exported as .csv and
+      // imported into MSFT Excel, Acces & Power BI. For now, we're just going to
+      // modify the data for export to solve issues related to returns and double-quotes
+      // In the future, we may choose to clean these values at the database level.
+
+      const columnsToClean = [
+        "rpt_street_name",
+        "rpt_street_desc",
+        "rpt_sec_street_desc",
+        "rpt_sec_street_name",
+      ];
+
+      columnsToClean.map(col => {
```

Comment:
Just noticed this console warning when switching back to VZV. I think this would go away if `.map()` was changed to `.forEach()` here.
`Line 114:  Expected to return a value in arrow function  array-callback-return`

Reply:
thanks for this tip, makes sense after looking more into the linter warning.